library/std/src/backtrace.rs

```diff
@@ -293,7 +293,7 @@ impl Backtrace {
         if !Backtrace::enabled() {
             return Backtrace { inner: Inner::Disabled };
         }
-        Backtrace::create(Backtrace::capture as usize)
+        Backtrace::create((Backtrace::capture as *mut ()).addr())
```

Comment:
AVR sadness function pointer cast

Reply:
😅 what does `*mut ()` even mean

Reply:
It's a pointer to the empty tuple. literally the type `()`, which is the default `void` for Rust, absent a better choice.